README.md

```diff
@@ -54,35 +54,42 @@ IPC uses [Fendermint](https://github.com/consensus-shipyard/fendermint) as the u
+With Docker installed, you can then `make install-infra` in the root of the `ipc` repo. This will clone the fendermint repo, build the docker image that you need to run subnets, and make the infrastructure scripts available in `./bin/ipc-infra/fendermint/infra` folder.
 
-<!-- In Unix-based systems, it is highly recommended to include your user in the `docker` group to avoid having to run many of the commands from this tutorial using `sudo`. You can achieve this running: -->
-<!-- ```bash -->
-<!-- sudo usermod -aG docker $USER -->
-<!-- newgrp docker -->
-<!-- ``` -->
+In Unix-based systems, it is highly recommended to include your user in the `docker` group to avoid having to run many of the commands from this tutorial using `sudo`. You can achieve this running:
+```bash
+sudo usermod -aG docker $USER
+newgrp docker
+```
 
-<!-- ## Connecting to a rootnet -->
+## Connecting to a rootnet
 
-<!-- You can deploy an IPC hierarchy from any compatible rootnet. The recommended option is to use Filecoin Calibration, but you can also deploy your own.  -->
+You can deploy an IPC hierarchy from any compatible rootnet. The recommended option is to use Filecoin Calibration, but you can also deploy your own.
 
-<!-- ### Option 1: Calibration -->
-<!-- Calibration is the primary testnet for Filecoin. It already hosts the IPC actors and can be used as a rootnet on which to deploy new subnets.  -->
+### Running a subnet in Calibration
+Calibration is the primary testnet for Filecoin. It already hosts the IPC actors and can be used as a rootnet on which to deploy new subnets.
 
-<!-- In order to use the IPC agent with Calibration we need to have access to a full node syncing with the network. The easiest way to achieve this is to use a [public RPC](https://docs.filecoin.io/networks/calibration/rpcs/). You also need the addresses of the deployed contracts The suggested configuration for the IPC agent is: -->
+In order to use the `ipc-cli` with Calibration we need to have access to a full node syncing with the network. The easiest way to achieve this is to use a [public RPC](https://docs.filecoin.io/networks/calibration/rpcs/). You also need the addresses of the deployed contracts.
 
-<!-- ``` -->
-<!-- # Default configuration for Filecoin Calibration -->
-<!-- [[subnets]] -->
-<!-- id = "/r314159" -->
-<!-- [subnets.config] -->
-<!-- gateway_addr = "0x5fBdA31a37E05D8cceF146f7704f4fCe33e2F96F" -->
-<!-- network_type = "fevm" -->
-<!-- provider_http = "https://api.calibration.node.glif.io/rpc/v1" -->
-<!-- registry_addr = "0xb505eD453138A782b5c51f45952E067798F4777d" -->
-<!-- ``` -->
+If it is the first time that you use your `ipc-cli`, to initialize cli configuration you can run `./bin/ipc-cli config init`. This will populate a new default config file in `~/.ipc/config.toml`.
+
+The suggested configuration for the IPC agent is:
```

Comment:
I still see `IPC agenet` here, is it correct?

Reply:
I left a few behind, good catch. Fixed.